hooks/command

```diff
@@ -186,15 +186,22 @@ else
   while true
   do
     set +e
-    pod_json=$(kubectl get pod "$pod_name" -o json)
-    init_container_status="$(echo "$pod_json" | jq ".status.initContainerStatuses[0].state.terminated.exitCode")"
-    if [[ -n "$init_container_status" && "$init_container_status" != "0" ]]; then
-      echo "Warning: init container failed with exit code $init_container_status, this usually indicates plugin misconfiguration or infrastructure failure"
-      status="$init_container_status"
-    else
-      status="$(echo "$pod_json" | jq ".status.containerStatuses[0].state.terminated.exitCode")"
-    fi
+    pod_json=$(kubectl get pod "$pod_name" -o json 2>&1)
+    get_pod_status=$?
     set -e
```

Comment:
the `set -e` should be at the end of the block similar to the previous behavior so we don't terminate the script prematurely